### DISCUSSION TITLE
授权设备之后又变回免费版

### DISCUSSION THREAD

**Discussion by f6rest** [getlantern/lantern-forum-cn#1032] [Q&A]
https://github.com/user-attachments/assets/023b172d-3897-4278-89d0-e4eb7c4aff3e

ipad版8.0.2（sdk7.6.107）可以正常使用，但是连接不稳定。mac的7.9.7（20241104.185109）版，windows的7.9.7（20241104.185026）版，客户端更新提示需要账号密码登录之后，在授权设备的时候，用授权码和邮箱都会提示授权成功，点击确认之后回到主页面时又变成了免费版。详情请查看附件视频。
目前win和mac都没有账号密码登录的窗口，只能用ipad授权。但是每次都授权失败，应该是改了登录方式导致的，请尽快修复一下

**Comment by MaxMKMax** [comment]
@f6rest 您好. 我已经帮您提交反馈给技术团队了.

**Reply by f6rest** [reply]
有进展了吗？

**Reply by f6rest** [reply]
@Mike777Khoo 现在还是登不上去，已经一个多月了，处理怎么样了？

**Comment by MaxMKMax** [comment]
@f6rest 您好. 早就已经提交上去了. 我再帮您跟进一下.

**Comment by MaxMKMax** [comment]
@f6rest 您好. 请您提供我一下支付单据和账号邮箱.

**Reply by f6rest** [reply]
@Mike777Khoo 
账号：auga3@126.com
我是去年续费的VIP，凭据应该是附图的那个。。这个号用了快10年了

**Reply by f6rest** [reply]
![98c2c9fd73467e827e4ec483ded7ffec](https://github.com/user-attachments/assets/871de310-51aa-46e8-a475-bc0250a0c11b)
